e2e/shared/tests.js

```diff
@@ -15,14 +16,22 @@ module.exports = {
 
   verifyFiles: (done) => {
     [
-      'app.js',
       'index.html',
-      'metadata.json',
-      'polyfills.js',
-      'vendor.js'
+      'metadata.json'
     ].forEach(file => {
       expect(fs.existsSync(path.resolve(common.tmp, 'dist', file))).toEqual(true);
     });
+
+    [
+      'app.*.js',
+      'polyfills.*.js',
+      'skyux.*.chunk.js',
+      'vendor.*.chunk.js'
```

Comment:
These files now include hashes to their file names, so we need to use `glob` instead.